docs/source/user-guide/tabular/tutorials/plot_static_predictions.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""
+Using static predictions
+***********************
+
+In some cases the model evaluation can take a long time i.e very large dataset.
+For this case you may calculate the predictions beforehand and pass them to the check/suite.
+
+* If the train dataset shares indexes with the test dataset we will add train/test prefixes.
+  This will cause the IndexTrainTestLeakage to not work.  
+"""
+
+#%%
+# Generate data & model
+# =====================
+
+from deepchecks.tabular.datasets.classification.iris import (
+    load_data, load_fitted_model)
+
+train_dataset, test_dataset = load_data()
+model = load_fitted_model()
+
+#%%
+# Set Up The Required Data
+# ======================
+# We will calculate the feature importance which is optional but will affect some displays and checks.
+#
+# We also calculating all the model predict and predict_proba results.
+
+from deepchecks.utils.features import calculate_feature_importance
+
+feature_importance, _ = calculate_feature_importance(model, test_dataset)
+
+train_predictions = model.predict(train_dataset.features_columns)
```

Comment:
Agree with @Nadav-Barak that there should also be an option to pass only proba and is should be explained and shown here